transactions/views.py

```diff
@@ -47,3 +52,29 @@ class TransactionDelete(DeleteView):
 
 class TransactionList(ListView):
     model = Transaction
+
+
+class TransactionDetail(DetailView):
+    model = Transaction
+
+
+class Transfer(FormView):
```

Comment:
dodaj walidację aby nie mozna było przelać z wallet_1 na wallet_1, zobacz jak to teraz działa